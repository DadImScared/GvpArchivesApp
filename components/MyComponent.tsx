import * as React from "react";
import { Text } from "react-native";

class MyComponent extends React.Component {
  render() {
    return (
        <Text>
          hi
        </Text>
    );
  }
}

export default MyComponent;
