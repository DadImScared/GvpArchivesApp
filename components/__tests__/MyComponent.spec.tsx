import * as React from "react";
import * as renderer from "react-test-renderer";

import MyComponent from "../MyComponent";

describe("MyComponent", () => {
  it("should render", () => {
    const tree = renderer.create(<MyComponent />);
    expect(tree).toMatchSnapshot();
  });

});
