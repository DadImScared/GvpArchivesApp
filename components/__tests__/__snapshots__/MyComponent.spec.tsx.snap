// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent should render 1`] = `
<div>
  hi
</div>
`;
